jest.config.js

```diff
@@ -6,12 +6,44 @@ const customJestConfig = {
   setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
   moduleDirectories: ['node_modules', '<rootDir>/', 'src'],
   testEnvironment: 'jest-environment-jsdom',
+  extensionsToTreatAsEsm: ['.ts'],
   moduleNameMapper: {
     lib$: '<rootDir>/src/lib/$1',
     'pages/(.*)$': '<rootDir>/src/pages/$1',
     'styles/(.*)$': '<rootDir>/src/styles/$1',
     'components/(.*)$': '<rootDir>/src/components/$1',
+    'react-markdown':
+      '<rootDir>/node_modules/react-markdown/react-markdown.min.js',
   },
 };
 
-module.exports = createJestConfig(customJestConfig);
```

Comment:
So all this weirdness is because `remark-directive` is an ESM-only module. By default Jest does not transform node_modules, so here we specify that we want it to transform `remark-directive` (along with all of its dependencies) before running.